ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/FieldCollection.java

```diff
@@ -584,6 +584,17 @@ public FieldCollectionBuilder<Type, StateType, Parent> label(String id, String v
       return this;
     }
 
+    public FieldCollectionBuilder<Type, StateType, Parent> label(String id, String value,
+                                                                 String showCondition, boolean showSummary) {
+      explicitFields.add(field(id)
```

Comment:
Could you add an example usage in `src/test/java/uk/gov/hmcts/reform/fpl/CCDConfig.java`

Reply:
@linusnorton  I included and updated required test expected files as well